packages/fxa-content-server/app/tests/spec/views/ready.js

```diff
@@ -119,6 +119,36 @@ describe('views/ready', function() {
       });
     });
 
+    it('renders `escapedEmailReadyText` as expected with `primary_email_verified` view type and param', () => {
+      windowMock.location.search = '?primary_email_verified=some@email.com';
+      sinon.stub(relier, 'isSync').returns(false);
```

Comment:
It might be good to run `relier.isSync.restore()` at the end of the test, in case someday someone in some other tests expects `relier.isSync` to have its original behavior.